repose-aggregator/functional-tests/spock-functional-test/src/test/configs/features/core/tracing/rewritetransid/system-model.cfg.xml

```diff
@@ -13,4 +13,5 @@
                       default="true"/>
         </destinations>
     </repose-cluster>
+    <tracing-header rewrite-header="true"/>
```

Comment:
I know this is contrary to what we decided for the story, but I may as well voice my concern.

I worry that the new behavior around the `tracing-header` element will confuse users. The line highlighted above shows why -- the `enabled` attribute is not shown be default. So if I were trying to turn the tracing heading off, my first thought might be that I can do so by removing the element. A reasonable thought, since that is the behavior in other configs.

Now that's all well and good, but what can be done about it? Well, I don't have a solution, but I do have an idea. Why not let the presence dictate whether or not to add the `tracing-header`? If we want the `tracing-header` to be added by default, that's fine, we ship a config which includes the element. The user still has to explicitly do *something* to disable the `tracing-header`, so in practical terms, we've set a default value. We also remove an attribute and keep the behavior for the element consistent with other elements in other configs.

Oh, I'd also consider renaming the element to `add-tracing-header` or `set-tracing-header` if the `enabled` attribute were to be removed.

Just my :moneybag: (there is two cents in there)
:two: :hankey: (two scents?)
:two: :email: (two sent)
:v: :nose: (two scents again?)
:trollface: (that's apparently a thing)